AerofoilSDL/ShaderCode/DrawQuadPaletteP.cpp

```diff
@@ -29,8 +29,16 @@
 
 namespace GpBinarizedShaders
 {
+	//macos does not support precision in OpenGLES2
```

Comment:
Handle this by changing the precision defs in Functions.h to an empty string instead of changing every shader.